Exception ignored in: <module 'collections.abc' from '/usr/lib/python3.6/collections/abc.py'>
Traceback (most recent call last):
  File "<string>", line 1, in <module>
KeyboardInterrupt
/usr/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: Cython module failed to patch module with custom type
  return f(*args, **kwds)
Using backend: pytorch
Traceback (most recent call last):
  File "pseudo_mini_batch_range_arxiv_sage.py", line 3, in <module>
    import dgl
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/__init__.py", line 18, in <module>
    from . import distributed
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/distributed/__init__.py", line 17, in <module>
    from .dist_graph import DistGraphServer, DistGraph, node_split, edge_split
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/distributed/dist_graph.py", line 20, in <module>
    from .partition import load_partition, load_partition_book
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/distributed/partition.py", line 12, in <module>
    from ..data.utils import load_graphs, save_graphs, load_tensors, save_tensors
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/data/__init__.py", line 29, in <module>
    from .rdf import AIFBDataset, MUTAGDataset, BGSDataset, AMDataset
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/data/rdf.py", line 12, in <module>
    import rdflib as rdf
  File "/home/cc/.local/lib/python3.6/site-packages/rdflib/__init__.py", line 184, in <module>
    from rdflib.graph import Dataset, Graph, ConjunctiveGraph
  File "/home/cc/.local/lib/python3.6/site-packages/rdflib/graph.py", line 244, in <module>
    from rdflib import plugin, exceptions, query
  File "/home/cc/.local/lib/python3.6/site-packages/rdflib/plugin.py", line 111, in <module>
    from pkg_resources import iter_entry_points
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 76, in <module>
    __import__('pkg_resources.extern.packaging.specifiers')
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/packaging/specifiers.py", line 630, in <module>
    _prefix_regex = re.compile(r"^([0-9]+)((?:a|b|c|rc)[0-9]+)$")
  File "/usr/lib/python3.6/re.py", line 233, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.6/sre_parse.py", line 855, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, 0)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 765, in _parse
    p = _parse_sub(source, state, sub_verbose, nested + 1)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 614, in _parse
    if not item or (_len(item) == 1 and item[0][0] is AT):
  File "/usr/lib/python3.6/sre_parse.py", line 159, in __len__
    def __len__(self):
KeyboardInterrupt
